FIX|TK8-38|Cluster Lifecycle

arush-sal opened the pull request.

Commit:
Add lifecycle functionality for scale and reset

Signed-off-by: Arush Salil <arush@Arushs-MacBook-Pro.local>

arush-sal committed:
Add lifecycle functionality for scale and reset

Signed-off-by: Arush Salil <me@aru.sh>

MuellerMH committed:
TK8-38 | FIX | rebase conflict develop

Signed-off-by: Manuel Müller <mueller.m.h@gmail.com>

MuellerMH committed:
TK8-38 | Fix | pull conflict

Signed-off-by: Manuel Müller <mueller.m.h@gmail.com>

MuellerMH reviewed (changes requested):
Please do not merge
There are some issues in this Request
for example:
internal/cluster/aws.go has still a merge conflict in the code
the new methods scale reset have the wrong path.

MuellerMH committed:
TK8-38|LiveCycle|Merge in new provisioner concept

Signed-off-by: Manuel Müller <mueller.m.h@gmail.com>

MuellerMH committed:
TK8-38 | Modify | Naming in RunPlaybook and fix parameters

Signed-off-by: Manuel Müller <mueller.m.h@gmail.com>

MuellerMH committed:
TK8-38 | Modify | fixing hosts file

not sure was happend there and how to use the scale
but it is not intuitive at the moment

Signed-off-by: Manuel Müller <mueller.m.h@gmail.com>
